### PR TITLE
data:manishd17

### PR DESCRIPTION
**Pull request by manishd17**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers
You all are doing an awesome work! Just love this community

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8162"></a>

### CODE REVIEW

File: data/manishd17.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+	"name": "Manish Jha",
+	"displayStatsPublic": true,
+	"type": "personal",
```
Comment (kumarsonsoff3) [inline data/manishd17.json:4]: We no longer need the `displayStatsPublic`, `type` keys. Please remove them from your `json`. 


```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/manishd17.json (hunk 1 above)
Comment (manishd17) [reply data/manishd17.json]: Ok, Thank you

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by manishd17** [comment]
How much time it'll take to reflect the changes?

**Comment by kumarsonsoff3** [comment]
> How much time it'll take to reflect the changes?

Your changes should already be live, usually after merge, it deploys automatically in about 5-10 minutes & changes got reflected after deployment. 

Your profile is available at: https://linkfree.io/manishd17

**Pull request merged**